src/components/Player/Player.js

```diff
@@ -6,21 +6,37 @@ import Search from './Search/Search';
 
 
 class Player extends Component {
-    render() {
-      return (
-        <div className="wrapper">
-            <div className='nav'>
-                <Link to="/counter">To the Counter</Link>
-                <Link to="/form">To the Increment form</Link>
-                {/* player */}
-                <Link to='/player'> Player </Link>
-            </div>
-            <Search />
-            <Information />
-            <Opsheet />
-        </div>
-      );
+  state = { 
+    opSheetItems: [], 
+  };
+
+  addItemToOpsheet = (name) => {
+    return () => {
+      this.setState(state => {
+        const opSheetItems = state.opSheetItems.concat(name)
+        console.log(opSheetItems);
+        return opSheetItems
```

Comment:
I'm only about 95% certain without running the code myself, but I suspect you likely want to return `{opSheetItems}` here, so as to update the `opSheetItems` object on state. Otherwise, I believe this will clobber all of the other values on `state` and in future runs, `state.opSheetItems` won't exist.